src/AppInstallerTestExeInstaller/main.cpp

```diff
@@ -39,8 +39,8 @@ path GenerateUninstaller(std::wostream& out, const path& installDirectory, const
 
     std::wofstream uninstallerScript(uninstallerPath);
     uninstallerScript << "@echo off\n";
-    uninstallerScript << "ECHO. >" << uninstallerOutputTextFilePath << "\n";
-    uninstallerScript << "ECHO AppInstallerTestExeInstaller.exe uninstalled successfully.\n";
+    uninstallerScript << "ECHO. >" << uninstallerOutputTextFilePath;

```

Comment:
nit: just curious is there a specific reason to move the \n to next line? For code style consistency within this file, we put \n to the line end.